subprojects/docs/src/docs/userguide/dep-man/03-controlling-transitive-dependencies/dependency_downgrade_and_exclude.adoc

```diff
@@ -63,6 +63,7 @@ For this reason, a good practice is that if you use _strict versions_, you shoul
 For example, `B` might say, instead of `strictly 1.0`, that it _strictly depends_ on the `[1.0, 2.0[` range, but _prefers_ `1.0`.
 Then if a consumer chooses 1.1 (or any other version in the range), the build will _no longer fail_ (constraints are resolved).
 
+[[forced_dependencies_vs_strict_dependencies]]
```

Comment:
Other section links use `sec:` not sure what that is used for

Reply:
Our internal link checker requires explicit section headers. But when we did not create one initially, I prefer making the implicit one explicit. That makes sure existing links still work, if they pointed at the `current` version of the docs.

Reply:
I think the question was why `sec:` is used sometimes as the prefix for a section header. There must have been some idea of a structure to that in the past when the first version of the user manual was created, but it is not used consistently (anymore). I would have used it here, but I think we don't have hard rules and we don't enforce it. We could also just drop it everywhere.

Reply:
Ok, maybe I was not clear in my answer:

* From what I have observed, and attempted to make consistent in the dependency management documentation rework:
  * `sec:` is used for highest level sections in a page (typically `==` level)
  * `sub:` is used for lower level sections in a page
* However in this case, the explicit id was not added when the section was created, so I did not add it as it risks breaking existing links out there.

And since I now have an internal link to the section, the explicit id is required.

So when adding a new section, you should always add an explicit id that uses the rule above. At least that's what I am attempting to do ... and may have failed in places.